src/app/containers/Include/index.jsx

```diff
@@ -42,6 +46,11 @@ const IncludeContainer = ({ html, type }) => {
     display: grid;
   `;
 
+  logger.info(INCLUDE_RENDERED, {
+    type,
+    html,
```

Comment:
I'm skeptical about having the `html` in the log output, as it would just be a large chunk of markup and not necessary info that might aid debugging. I think just the `type` as this ensures includes render fine. Feel free to object.

Reply:
yeah, that makes sense to me, just wasn't sure if the type alone was enough info? @rhenshaw56

Reply:
I agree that removing `html` would be best. 

I think it'd be useful to have the `includeUrl` and the `pageUrl` listed. These are not currently available in the containers, but we could add `pageUrl` via the Request Context.  
This can happen in a separate PR since I believe it'd involve work in a lot of different files. 
I have a more narrowly defined issue that captures adding `pageUrl` to the logs within the `getInitialData` directory here: https://github.com/bbc/simorgh/issues/6784
I will update that issue to include logging within containers such as this one.

Reply:
> I think it'd be useful to have the includeUrl and the pageUrl listed. These are not currently available in the containers

I just remembered the `includeUrl` is available as a `href` prop in the container because it's passed in from the pre-processor but because it's has no obvious need so it seem like it's not there. Will be good to add it as well

![Screen Shot 2020-06-05 at 09 59 12](https://user-images.githubusercontent.com/25608386/83857704-45790e00-a713-11ea-83dd-bc1aa393f502.png)